gui/packages/ubuntupro/lib/constants.dart

```diff
@@ -12,3 +14,5 @@ const kVersion = String.fromEnvironment(
   'UP4W_FULL_VERSION',
   defaultValue: 'Dev',
 );
+
+const kConfirmColor = Color(0xFF0E8420);
```

Comment:
Maybe we should look at replacing this custom color with a built-in color in the future. It's one of the only custom colors we use in the app (other than `Color(0xFFE6F8E8)` in one place on the subscription status page).

Reply:
Interestingly this constant is the value of `YaruColors.success`.
https://ubuntu.github.io/yaru.dart/
https://github.com/ubuntu/yaru.dart/blob/main/lib/src/themes/colors.dart#L25-L39

I think we should use the Yaru value deliberately now that's available.